templates/index.html

```diff
@@ -12,7 +12,18 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        {% for item in items %}
+        <li>{{item.title}} - {{item.status}}</li>
+        {% endfor %}
       </ul>
     </div>
   </div>
+  <form action="/add_Item" method="post">
+    <li>
+    <label for="title">title:</label>
```

Comment:
The label should be indented one more level to make the nesting clearer (the `label` and `input` are both inside the `li` tag